test/message/testcfg.py

```diff
@@ -33,6 +33,7 @@
 from functools import reduce
 
 FLAGS_PATTERN = re.compile(r"//\s+Flags:(.*)")
+FLAGS_ONLY_PATTERN = re.compile(r"//\s+FlagsOnly")
```

Comment:
I have concerns about modifying the Python test runner to accommodate a change to JS code. Why is this needed? If it's not possible to add a message test without changing the Python runner, then I think we should add a parallel test instead.

Reply:
TBH, I picked this from @MoLow, and without this flag, these tests are failing and I can't explain why.

>  then I think we should add a parallel test instead.

Should add other parallel tests than the one already there for the parser?

Reply:
It looks like these are the errors. Not sure though!
![image](https://user-images.githubusercontent.com/1699357/197050873-fef71ff6-9fa8-4fb0-9da7-d8a10841eab9.png)

Reply:
I think you'll have an easier time if you do this:

- Undo the changes in testcfg.py.
- Remove the message tests here.
- Add a test in `test/parallel/test-runner-cli.js` that runs the command `node --test test/fixtures/test-runner/index.test.js test/fixtures/test-runner/nested.js test/fixtures/test-runner/invalid-tap.js` and verifies that output for passing tests is included.

After that, I think this will be ready to merge.

Reply:
+1.

Reply:
Will do 👍